src/test/daml/Daml/Finance/Account/Test/Controllers.daml

```diff
@@ -101,14 +128,22 @@ ownerControlledWithAutoApproval = script do
     setupInitialState tp (Fungible.Factory with provider = custodian; observers = pp) (M.toList pp)
       OwnerWithAutoApproval
 
+  -- Build index table
+  idx <- Holding.buildIndexTableForTransfer [custodian] issuerHoldingCid investorAccount
+
   -- Custodian can't transfer.
   submitMultiMustFail [custodian] [publicParty] do
     exerciseCmd (coerceInterfaceContractId @Transferable.I issuerHoldingCid)
-      Transferable.Transfer with actors = S.fromList [custodian]; newOwnerAccount = investorAccount
+      Transferable.Transfer with
+        actors = S.fromList [custodian]; newOwnerAccount = investorAccount; idx
+
+  -- Build index table
+  idx <- Holding.buildIndexTableForTransfer [issuer, publicParty] issuerHoldingCid investorAccount
```

Comment:
no need

Reply:
this is done to check proper contract visibility